WalletWasabi/WabiSabi/Crypto/WabiSabiClient.cs

```diff
@@ -101,7 +101,8 @@ public ZeroCredentialsRequestData CreateRequestForZeroAmount()
 		{
 			// Make sure we request always the same number of credentials
 			var credentialAmountsToRequest = amountsToRequest.ToList();
-			var missingCredentialRequests = NumberOfCredentials - amountsToRequest.Count();
+			var expectedNumberOfCredentials = amountsToRequest.Any() ? NumberOfCredentials : 0;
```

Comment:
perhaps it will be a little cleaner to make this variant private, and then have two overloads:

1. one that enforces that it has to have a length == `NumberOfCredentials`
2. one that takes no `amountsToRequest` argument at all

Reply:
Done